subgraph/src/utils/helper.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-inferrable-types */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
Please, fix the errors and don't disable linter.
For no-shadow you need to add eslint configuration and add BigInt to exceptions.
no-underscore-dangle - can be left as is.